store/evm_aux/config.go

```diff
@@ -0,0 +1,28 @@
+package evmaux
+
+type EvmAuxStoreConfig struct {
+	// DBName defines database file name
+	DBName string
+	// DBBackend defines backend EVM store type
+	// available backend types are 'goleveldb', or 'cleveldb'
+	DBBackend string
+	// MaxReceipts defines the maximum number of EVM tx receipts stored in EVM auxiliary store
+	MaxReceipts uint64
```

Comment:
And what happens when this is zero?

Reply:
if set to 0, no receipts are persisted in EVM auxiliary store, will add this comment